invertedai/utils.py

```diff
@@ -563,6 +575,211 @@ def inside_fov(center: Point, initialize_fov: float, point: Point) -> bool:
     response.agent_attributes = agent_attributes
     return response
 
+def _get_drivable_area_ratio(centers,location,agent_density,scaling_factor):
+    try:
+        from cv2 import (
+            cvtColor,
+            COLOR_BGR2GRAY
+        )
+
+    except ImportError as e:
+        iai.logger.error(
+            "Calculating drivable area requires cv2, which was not found."
+        )
+        raise e
```

Comment:
I don't understand why this is needed at all. It seems like we're only using it to count the black pixels, which are zeros in any representation, so why do we need to import a color converter from this heavy package that's otherwise not our dependency?

Reply:
Something similar is done in the Image object in common.py so I figured this was the acceptable way to do so. I can find a lighter-weight colour-to-grayscale converter or just do it manually in the code.

Reply:
I see, so you can't decode the image at all without cv2. I guess we can leave this here then.

Reply:
Something similar is done in the Image object in common.py so I figured this was the acceptable way to do so. I can find a lighter-weight colour-to-grayscale converter or just do it manually in the code.

Reply:
I added a manual grayscale conversion using Luminosity method and added a comment saying as much for future reference.